tests/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -684,7 +683,6 @@
 # For details see: https://github.com/typeddjango/django-stubs/issues/1022
 -   case: from_queryset_custom_auth_user_model
     # Cache needs to be disabled to consistently reproduce the bug
```

Comment:
This comment should be removed?

Reply:
Or better `disable_cache: true` restored, good catch!

Reply:
Yes my bad, I forgot to revert the disable_cache lines when cleaning up. Can send a pr later for that